src/testUtils/mocks/entities.ts

```diff
@@ -693,2143 +1470,315 @@ export const mockKnownPermissionGroupModule = (path: string) => (): Record<strin
-  }
-
-  return securityTokenGetIdentifiersStub;
-}
-
-/**
- * @hidden
- * Retrieve the stub of the `SecurityToken.Transfers.canTransfer` method
- */
-export function getSecurityTokenTransfersCanTransferStub(status?: TransferStatus): SinonStub {
-  if (status) {
-    return securityTokenTransfersCanTransferStub.resolves(status);
-  }
-
-  return securityTokenTransfersCanTransferStub;
-}
-
-/**
- * @hidden
- * Retrieve the stub of the `SecurityToken.transferRestictions.count.get` method
- */
-export function getSecurityTokenTransferRestrictionsCountGetStub(
-  restrictions?: ActiveTransferRestrictions<CountTransferRestriction>
-): SinonStub {
-  if (restrictions) {
-    return securityTokenTransferRestrictionsCountGetStub.resolves(restrictions);
-  }
-
-  return securityTokenTransferRestrictionsCountGetStub;
-}
-
-/**
- * @hidden
- * Retrieve the stub of the `SecurityToken.transferRestictions.pecentage.get` method
- */
-export function getSecurityTokenTransferRestrictionsPercentageGetStub(
-  restrictions?: ActiveTransferRestrictions<PercentageTransferRestriction>
-): SinonStub {
-  if (restrictions) {
-    return securityTokenTransferRestrictionsPercentageGetStub.resolves(restrictions);
-  }
-
-  return securityTokenTransferRestrictionsPercentageGetStub;
-}
-
-/**
- * @hidden
- * Retrieve the stub of the `SecurityToken.corporateActions.getAgents` method
- */
-export function getSecurityTokenCorporateActionsGetAgentsStub(agent?: Identity): SinonStub {
-  if (agent) {
-    return securityTokenCorporateActionsGetAgentsStub.resolves(agent);
-  }
-
-  return securityTokenCorporateActionsGetAgentsStub;
-}
-
-/**
- * @hidden
- * Retrieve the stub of the `SecurityToken.corporateActions.getDefaultConfig` method
- */
-export function getSecurityTokenCorporateActionsGetDefaultConfigStub(
-  defaults?: Partial<CorporateActionDefaultConfig>
-): SinonStub {
-  if (defaults) {
-    return securityTokenCorporateActionsGetDefaultConfigStub.resolves(defaults);
-  }
-
-  return securityTokenCorporateActionsGetDefaultConfigStub;
-}
-
-/**
- * @hidden
- * Retrieve the stub of the `SecurityToken.permissions.getGroups` method
- */
-export function getSecurityTokenPermissionsGetGroupsStub(
-  groups?: Partial<ResultSet<PermissionGroup>>
-): SinonStub {
-  if (groups) {
-    return securityTokenPermissionsGetGroupsStub.resolves(groups);
-  }
-
-  return securityTokenPermissionsGetGroupsStub;
-}
-
-/**
- * @hidden
- * Retrieve the stub of the `SecurityToken.permissions.getAgents` method
- */
-export function getSecurityTokenPermissionsGetAgentsStub(
-  agents?: Partial<AgentWithGroup>[]
-): SinonStub {
-  if (agents) {
-    return securityTokenPermissionsGetAgentsStub.resolves(agents);
-  }
-
-  return securityTokenPermissionsGetAgentsStub;
-}
-
-/**
- * @hidden
- * Retrieve an Authorization Request instance
- */
-export function getAuthorizationRequestInstance(
+export const getAuthorizationRequestInstance = function (
```

Comment:
(style) I think if we are assigning to a variable and exporting that, then these should be arrow functions.

I think its OK to leave as `export function funcName() {` as well, but variable assignment with function keyword doesn't seem idiomatic to me.